Refactor minor fixes (#275)
AlphaGenes/AlphaPeel: docs/source/usage.rst

The ``.geno_prob.txt`` file contains *genotype probabilities* for each individual.

There are three lines per individual, corresponding to probabilities for
``aa``, ``aA`` or ``Aa``, and ``AA`` genotypes.
The first value in each line is the individual ID.
The remaining values are genotype probabilities at each locus.

> When working with the X chromosome for male individual, the first line and 

Reviewer: This won’t be clear to most as it’s too cryptic.

I suggest the following.
1) Take the X example input from the above (there id1 and id3 are males…)
2) Get genotype probabilities for the example
3) Create a separate output example for the X chromosome below the autosomal chromosome 
4) Add this text above that example
5) Mention that for females the interpretation is as for an autosomal chromosome below, but for males …

I will then suggest edits

  id4 0.0001 0.0001 0.9999 0.0000
  id4 0.0000 0.9999 0.0000 0.0000

> When working with the X chromosome, for female individual, the interpretation

Reviewer: ```suggestion
When working with the X chromosome, for a female individual, the interpretation
```

> is as for an autosomal chromosome above. For male individual, the first line and 

Reviewer: ```suggestion
is as for an autosomal chromosome above. For a male individual, the first line and 
```

> the last line corresponding to ``a`` and ``A`` respectively, with the middle

Reviewer: ```suggestion
the last line respectively correspond to ``a`` and ``A`` genotypes, with the middle
```

> line used as a placeholder. 

Reviewer: ```suggestion
line is used as a placeholder. 
```

> line is used as a placeholder. 

Reviewer: ```suggestion
line is a placeholder. 
```

  id4 0.0000 0.0000 0.7088 0.0000
  id4 0.0000 0.9999 0.0000 0.0000

> When working with the X chromosome, for female individual, the interpretation

Reviewer: Apply the same edits as above

is as for an autosomal chromosome above. For a male individual, the first line and 
the last line respectively correspond to ``a`` and ``A`` genotypes, with the middle 

> two lines used as placeholders.

Reviewer: ```suggestion
two lines are placeholders.
```
